src/components/CartItems.jsx

```diff
@@ -0,0 +1,102 @@
+import ReactStarsRating from "react-awesome-stars-rating";
+import { addToCart, removeFromCart } from "../redux/actions";
+import { useNavigate } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import { Flex, Text, Button } from "@sparrowengg/twigs-react";
+import EmptyCartUI from "./EmptyCart";
+const CartItems = () => {
+  const cart = useSelector((state) => state.cart);
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  return (
+    <>
+      {cart.length === 0 ? (
+        // <Flex css={{ width: "100%" }}>
+        <EmptyCartUI />
+      ) : (
+        // </Flex>
+        <Flex flexDirection="column" css={{ overflow: "scroll" }}>
+          <Text
+            css={{
+              marginBlock: "$15",
+              textAlign: "center",
+              fontSize: "$3xl",
+            }}
+          >
+            Review Your Bag Items
+          </Text>
+          <Flex
+            flexDirection="column"
+            css={{ overflow: "scroll", gap: "$15", padding: "$10" }}
+          >
+            {cart.map((ele) => (
+              <Flex
+                key={ele.id}
+                css={{
+                  background: "#ffffff",
+                  borderRadius: "20px",
```

Comment:
@arun-121 can you checkout our theming values

Reply:
changed to theming values